### PR TITLE
Improved Build Guide

### LINKED ISSUE #21: Missing step in building guide for windows
**File:** _posts/2019-09-10-guide_building_the_game.md

Building in Windows requires the C++ Desktop component ([this](https://cdn.discordapp.com/attachments/235111289435717633/631543356924624896/unknown.png)) which is not mentioned in the steps. It may also require other components as I installed a few other extra things, so it might be a good idea to double check all that.

Also what's with [this](https://github.com/momentum-mod/game/wiki)? Found this page before finding the docs site and it led me to think you still need a key to play the latest ver. Might as well trash it now that you have an actual (and recent) wiki.

### PR DESCRIPTION
**Pull request by Flipybitz**
Closes #21 
Improved the wording and reduced ambiguity with certain parts of the guide. Removed the OSX part of building the game.

Also removed text saying "Please reach out to Gocnak explaining why you'd like a key" because obvious reasons.

download
download (1)
download (2)
download (3)
download (4)

### CODE REVIEW

**Comment by Flipybitz** [comment]
> This looks like a typo, according to this commit you want `build6.png`.

Already spotted it and fixed it

File: _posts/2019-09-10-guide_building_the_game.md [hunk 1]
```diff
@@ -32,9 +31,9 @@ Go to the [***Main Page***](https://github.com/momentum-mod/game) and click ***C
 
 <img src="/assets/images/build_guide/build0.png" alt="" style="display: block; margin: auto;">
 
-You should have cloned the `develop` branch.  
+You should clone the `develop` branch.
```
Comment (Gocnak) [inline _posts/2019-09-10-guide_building_the_game.md]: `checkout`, not clone

File: _posts/2019-09-10-guide_building_the_game.md [hunk 2]
```diff
@@ -47,41 +46,50 @@ It may also be called `origin/develop`:
...
+When installing Visual Studio, make sure this version is checked:
+
+<img src="/assets/images/build_guide/build6.png" alt="" style="display: block; margin: auto;">
+    </p>
+</div>
+
 Follow [this step on the Valve Dev Wiki](https://developer.valvesoftware.com/wiki/Source_SDK_2013#Unable_to_find_RegKey_for_.vcproj_files_in_solutions_.28Windows.29) to add a missing registry key to your computer.
 
-Get a Git client like [***GitHub Desktop***](https://desktop.github.com/) if you haven't already and then clone this repository.
+Get a Git client like [***GitHub Desktop***](https://desktop.github.com/) if you haven't already and then clone the develop branch of the repository.
```
Comment (Gocnak) [inline _posts/2019-09-10-guide_building_the_game.md]: You can't clone a branch, you clone a repository. You check out branches. This change makes less sense than what was originally there.

File: _posts/2019-09-10-guide_building_the_game.md [hunk 3]
```diff
@@ -47,41 +46,50 @@ It may also be called `origin/develop`:
...
 Run `creategameprojects.bat` **as administrator**.
 
 <img src="/assets/images/build_guide/build3.png" alt="" style="display: block; margin: auto;">
 
+This script will create a symlink of your `mp/game/momentum` folder into the `MomentumDev` folder automatically, which allows you to test changes immediately and run the debugger from Visual Studio.
+
 <div class="note info">
     <p>
-		If you can't run it as administrator, start <code>cmd.exe</code> as Administrator, then change directory to the Git clone, and run it from <code>cmd.exe</code>.
+		If you can't run it as administrator, start <code>cmd.exe</code> as Administrator, then `cd` to the develop repository and run it from <code>cmd.exe</code>.
```
Comment (Gocnak) [inline _posts/2019-09-10-guide_building_the_game.md]: What is a develop repository? This doesn't make sense.
```suggestion
		If you can't run it as administrator, start <code>cmd.exe</code> as Administrator, then `cd` to the location of the file on disk and run it from <code>cmd.exe</code>.
```

File: _posts/2019-09-10-guide_building_the_game.md [hunk 4]
```diff
@@ -47,41 +46,50 @@ It may also be called `origin/develop`:
...
 
-If you downloaded Momentum Mod from Steam, create an extra folder somewhere called `MomentumDev`, and copy everything from your `steamapps/common/Momentum Mod` folder to it. Then delete the `momentum` folder from `MomentumDev`.
+If you've downloaded Momentum Mod from Steam, create an extra folder somewhere called `MomentumDev`, and copy everything from your `steamapps/common/Momentum Mod` folder to it. Delete the `momentum` folder from `MomentumDev`, as this will be symlinked later.
 
 Otherwise if you downloaded the `MomentumDev` folder from Google Drive, extract it somewhere.
 
-Edit the `creategameprojects.ps1` file to point the `$path` variable to the full path to `MomentumDev`.  
-
-This script will create a symlink of your `mp/game/momentum` folder into the `MomentumDev` folder automatically for you, which allows you to test changes immediately and run the debugger from Visual Studio.
+Edit `mp/src/creategameprojects.ps1` inside the develop branch to point the `$path` variable to the path where your `MomentumDev` folder is.
```
Comment (Gocnak) [inline _posts/2019-09-10-guide_building_the_game.md]: The file exists inside every branch.
```suggestion
Edit `mp/src/creategameprojects.ps1` to point the `$path` variable to the path where your `MomentumDev` folder is.  
```

File: _posts/2019-09-10-guide_building_the_game.md [hunk 5]
```diff
@@ -47,41 +46,50 @@ It may also be called `origin/develop`:
...
 
 ## Windows
-Download Momentum Mod on Steam or grab the `MomentumDev` folder from [***Google Drive***](https://drive.google.com/file/d/115xT5IIN-CimKCZQho3xj7uIcYFMSZXm/view).
+Download Momentum Mod on Steam, or grab the `MomentumDev` folder from [***Google Drive***](https://drive.google.com/file/d/115xT5IIN-CimKCZQho3xj7uIcYFMSZXm/view).
 
 Download [***Visual Studio 2019***](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16).
 
+<div class="note info">
+    <p>
+When installing Visual Studio, make sure this version is checked:
```
Comment (Gocnak) [inline _posts/2019-09-10-guide_building_the_game.md]:
```suggestion
When installing Visual Studio, make sure this environment is checked:
```

File: _posts/2019-09-10-guide_building_the_game.md [hunk 6]
```diff
@@ -47,41 +46,50 @@ It may also be called `origin/develop`:
...
+
+<img src="/assets/images/build_guide/build6.png" alt="" style="display: block; margin: auto;">
```
Comment (Gocnak) [inline _posts/2019-09-10-guide_building_the_game.md]: This image does not give a lot of context on where people can find it. I would suggest using a bigger picture that shows this tool box highlighted.

**Comment by Gocnak** [comment]
Closed in favor of #39

**Pull request closed without merging**